packages/m/mvfst/xmake.lua

```diff
@@ -19,6 +19,7 @@ package("mvfst")
                          "-DCMAKE_CXX_STANDARD=17"}
         table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
         table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
+        table.insert(configs, "-DCMAKE_POSITION_INDEPENDENT_CODE=" .. (package:config("shared") and "ON" or "OFF"))
```

Comment:
this is already automatically passed by xmake.
see modules/package/tools/cmake.lua:347
```lua
    if package:config("pic") ~= false then
        table.insert(configs, "-DCMAKE_POSITION_INDEPENDENT_CODE=ON")
    end
```

Reply:
yes, config is not inherited (well except for runtimes), setting a package as shared doesn't set its dependencies as shared as well.